rockcraft.yaml

```diff
@@ -0,0 +1,46 @@
+name: charmed-mysql # the name of your ROCK
+base: ubuntu:22.04 # the base environment for this ROCK
+version: '0.8.22-1' # just for humans. Semantic versioning is recommended
+summary: Charmed MySQL ROCK OCI # 79 char long summary
+description: |
+    MySQL built from the official ubuntu MySQL package 
+    from the Ubuntu repository and further installs 
+    mysql-shell.  For more information on ROCKs, visit 
+    the https://github.com/canonical/rockcraft.
+license: Apache-2.0 # your application's SPDX license
+cmd: 
+    - /usr/bin/setpriv
+    - --clear-groups
+    - --reuid
+    - mysql
+    - --regid
+    - mysql
+    - -- 
+    - /usr/sbin/mysqld
+platforms: # The platforms this ROCK should be built on and run on
+    amd64:
+
+parts:
+    mysql-repo:
+        plugin: nil
+        override-pull: |
+            apt-get update
+            apt-get install -y wget lsb-release gnupg
+            wget -q https://dev.mysql.com/get/mysql-apt-config_0.8.22-1_all.deb \
+                -O /tmp/mysql-apt-config_0.8.22-1_all.deb
+            dpkg -i /tmp/mysql-apt-config_0.8.22-1_all.deb
+            apt-get update
+    mysql-deb:
+        plugin: nil
+        after: [mysql-repo]
+        stage-packages:
+            - mysql-shell
+            - mysql-server-8.0
+            - util-linux
```

Comment:
thoughts on adding xtrabackup to the rock (as it will very shortly be needed)? it can be added in a separate PR as well

Reply:
There is already an open PR for it :)